transformer_engine/common/fused_attn/fused_attn_f16_arbitrary_seqlen.cu

```diff
@@ -77,6 +79,13 @@ void fused_attn_arbitrary_seqlen_fwd_impl(
   }
   auto cudnn_runtime_version = cudnnGetVersion();
 
+  // We choose between 32-bit and 64-bit offsets depending on need.
+  // This allows us to support older cuDNN runtimes gracefully.
+  const NVTE_QKV_Layout_Group layout_group = nvte_get_qkv_layout_group(layout);
+  const DType ragged_offset_type = ç(layout_group, h, hg, s_q, s_kv, d_qk, d_v);
```

Comment:
Typo for the function name?

Reply:
Or maybe this is not the latest version of your code? :)

Reply:
Fixed